jina/serve/runtimes/gateway/request_handling.py

```diff
@@ -61,17 +62,35 @@ def __init__(
                 labelnames=('runtime_name',),
             ).labels(runtime_name)
 
+            self._failed_requests_metrics = Gauge(
+                'number_of_failed_requests',
+                'Number of failed requests',
+                registry=metrics_registry,
+                namespace='jina',
+                labelnames=('runtime_name',),
+            ).labels(runtime_name)
+
+            self._successful_requests_metrics = Gauge(
```

Comment:
Same : I believe this should be a Counter and not a Gauge since the number cannot decrease